docs/ABI/TypeMetadata.rst

```diff
@@ -122,19 +122,21 @@ In addition to the `common metadata layout`_ fields, function metadata records
 contain the following fields:
 
 - The function flags are stored at **offset 1**, information contained by flags
-  includes 'throws' indicator (8 bits), metadata convention (8 bits)
-  and number of arguments (remaining N bits).
-- The **argument vector** begins at **offset 2** and consists of pointers to
-  metadata records of the function's arguments.
-- A reference to the **result type** metadata record is stored after the end
-  of **argument vector**. If the function has multiple returns, this references a
-  `tuple metadata`_ record.
-
-  A pointer to each argument's metadata record will be appended separately,
-  the lowest bit being set if it is **inout**. Because of pointer alignment,
-  the lowest bit will always be free to hold this tag.
-
-  If the function takes no arguments, **argument vector** is going to be empty.
+  includes flags - 'throws' bit, parameter flags bit (8 bits), function convention (8 bits)
+  and number of parameters (16 bits).
```

Comment:
Why are you saying that the parameter flags bit is 8 bits?

Reply:
Let me re-phrase that a bit if that's unclear - it should be - includes flags (8 bits) which consists of 'throws' bit and 'parameter flags' bit, function convention (8 bit) etc.